### PR TITLE
ACCEPTANCE: Get All Posts

### LINKED ISSUE #40: ACCEPTANCE: GET ALL Posts
Learn more about acceptance tests here:
https://www.youtube.com/watch?v=WWN-9ahbdIU

### PR DESCRIPTION
**Pull request by cacti79**
Closes #40

### CODE REVIEW

**Comment by cacti79** [comment]
Closes #40

File: Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
```
Comment (hassanhabib) [inline Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.Logic.cs:1]: Add file header here.

Comment (hassanhabib) [reply Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.Logic.cs:1]: Do the same for the other files.

File: Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+        private Filler<Post> CreatePostFiller()
+        {
+            var now = DateTimeOffset.UtcNow;
+            var filler = new Filler<Post>();
+
+            filler.Setup()
+                .OnProperty(post => post.CreatedDate).Use(now)
+                .OnProperty(post => post.UpdatedDate).Use(now)
+                .OnType<DateTimeOffset>().Use(GetRandomDateTimeOffset());
```
Comment (hassanhabib) [inline Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.cs:57]: This one is surprising to me. Is our date validations established?

Comment (cacti79) [reply Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.cs:62]: @hassanhabib 
I check Otriples as a refrence, and the code there is the same,
can you help me?

File: Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+using Xunit;
+
+namespace Taarafo.Core.Tests.Acceptance.Apis.Posts
+{
+    [Collection(nameof(ApiTestCollection))]
+    public partial class PostsApiTests
+    {
+        private readonly ApiBroker apiBroker;
+
+        public PostsApiTests(ApiBroker apiBroker)
```
Comment (ShriHumrudha) [inline Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.cs:21]: Single line code so remove the scope and have =>

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.Logic.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+namespace Taarafo.Core.Tests.Acceptance.Apis.Posts
+{
+    public partial class PostsApiTests
+    {
+        [Fact]
+        public async Task ShouldGetAllPostsAsync()
+        {
+            // given
+            List<Post> randomPosts = await CreateRandomPostedPostsAsync();
+
```
Comment (hassanhabib) [inline Taarafo.Core.Tests.Acceptance/Apis/Posts/PostsApiTests.Logic.cs:22]: Break the above line and keep the empty line here.

**Pull request closed without merging**